base/applications/control/control.c

```diff
@@ -99,6 +138,28 @@ wWinMain(HINSTANCE hInstance,
     else if (!_wcsicmp(lpCmdLine, L"userpasswords"))   return RunControlPanel(L"nusrmgr.cpl");       /* Graphical User Account Manager */
     else if (!_wcsicmp(lpCmdLine, L"userpasswords2"))  return RUNDLL(L"netplwiz.dll,UsersRunDll");   /* Dialog based advanced User Account Manager */
 
+    /* https://learn.microsoft.com/en-us/windows/win32/shell/executing-control-panel-items#windows-vista-canonical-names */
+    argv = CommandLineToArgvW(lpCmdLine, &argc);
+    if (argv)
+    {
+        UINT argi = 0;
+        HRESULT hr = -1;
+        if (argc >= 2 && IsSwitch(L"name", argv[argi + 0]))
+        {
+            LPCWSTR pszPage = NULL;
+            if (argc >= 4 && IsSwitch(L"page", argv[argi + 2]))
+            {
```

Comment:
You expect here that the arguments are given in the order: `/name XXXX`, followed by `/page YYYY` (as in the given example `"%systemroot%\system32\control.exe /name canonicalName /page pageName`).
What happens on Windows if the page and name arguments are specified in reverse order (e.g. `control.exe /page pageName /name canonicalName` ) ?

Reply:
> What happens on Windows if the page and name arguments are specified in reverse order

Nothing at all.

`control /page 1 /name Microsoft.DateAndTime` does not work while `control /name Microsoft.DateAndTime /page 1` of course works.